datacenter/ucp/2.2/guides/admin/backups-and-disaster-recovery.md

```diff
@@ -14,11 +14,14 @@ The next step is creating a backup policy and disaster recovery plan.
 ## Backup policy
 
 As part of your backup policy you should regularly create backups of UCP.
+DTR is backed up independently.
```

Comment:
What do you think about adding a [table like this](https://docs.docker.com/datacenter/dtr/2.3/guides/admin/backups-and-disaster-recovery) where we make clearer what is backed-up and what is not?

And probably after the table we can enumerate the order of operations:
1. [Backup your swarm](https://docs.docker.com/engine/swarm/admin_guide/#back-up-the-swarm)
2. Backup UCP
3. Backup DTR

Reply:
I added a table and took a guess at which data UCP backs up, judging by the named volumes. Somebody should probably check this. ;)